sdk-workflows/src/main/java/io/dapr/workflows/WorkflowContext.java

```diff
@@ -514,4 +515,19 @@ default void continueAsNew(Object input) {
    *                                  history, otherwise {@code false}
    */
   void continueAsNew(Object input, boolean preserveUnprocessedEvents);
+
+  /**
+   * Create a new UUID that is safe for replay within a workflow.
+   *
+   * <p>
+   * The default implementation of this method creates a name-based UUID
+   * using the algorithm from RFC 4122 §4.3. The name input used to generate
+   * this value is a combination of the workflow instance ID and an
+   * internally managed sequence number.
+   *</p>
+   * @return a deterministic UUID
+   */
+  default UUID newUuid() {
```

Comment:
do we need the `newUuid()` as part of the Workflow API contract? I am seeing this in not being used in this example.